modules/backend/layouts/_custom_styles.php

```diff
@@ -1,6 +1,8 @@
 <?php
+
     use Backend\Models\BrandSetting;
     use Backend\Models\EditorSetting;
+
```

Comment:
Do we have to have this whitespace? It doesn't add much IMO

Reply:
No we don't, probably just a tweak to the config for one of the rules. I'll have a look.

Reply:
I've added an exception for that rule for partials, since changing the config changes it across the board and would allow the following (which looks wonky IMO):

```php
<?php
namespace Winter\Storm\Classes;
use Illuminate\Support\Facades\DB;
class Something extends SomethingElse
{
...
```